lib/blocktrain/aggregation.rb

```diff
@@ -60,8 +60,7 @@ def aggs
           date_histogram: {
             field: 'timeStamp',
             interval: @interval,
-            pre_zone: '+01:00',
-            pre_zone_adjust_large_interval: true,
+            time_zone: '+01:00',
```

Comment:
It's because all the times are stored as UTC in the Elasticsearch DB. Although, that said, how this allows for daylight savings, I don't know